feature-nft-impl/src/main/java/io/novafoundation/nova/feature_nft_impl/data/source/providers/uniques/UniquesNftProvider.kt

```diff
@@ -121,6 +146,27 @@ class UniquesNftProvider(
         }
     }
 
+    override suspend fun subscribeNftOwnerAccountId(
+        subscriptionBuilder: StorageSharedRequestsBuilder,
+        nftLocal: NftLocal
+    ): Flow<AccountId?> {
+        return remoteStorage.query(nftLocal.chainId) {
+            val storage = runtime.metadata.uniques().storage("Asset")
+            val key = storage.storageKey(
+                runtime,
+                nftLocal.collectionId.toBigInteger(),
+                nftLocal.instanceId?.toBigInteger()
+            )
+            try {
+                subscriptionBuilder.subscribe(key)
+                    .map { bindNftOwnerAccountId(it.value, runtime) }
+                    .flowOn(Dispatchers.IO)
+            } catch (e: Exception) {
+                emptyFlow()
+            }
+        }
```

Comment:
Once changes from `develop` are merged, we can clean this piece and pass subscription builder to `remoteStorage`:

`remoteStorage.subscribe(chainId, subscriptionBuilder) {  ... }`